### DISCUSSION TITLE
Missing link to host_authorization plugin on roda website main documentation page

### DISCUSSION THREAD

**Discussion by loranger32** [jeremyevans/roda#294] [General]
Hi,

I was looking for the documentation of the `host_authorization` plugin (added in 3.43), but I realized there is no link to it on the main documentation page of the Roda website.

If it's not intentional, I guess an addition like this
```html
<li><a href="rdoc/classes/Roda/RodaPlugins/HostAuthorization.html"></a>: Allows configuring an authorized host or an array of authorized hosts.</li>
```
in `roda/www/pages/documentation.erb` under the "Request/Response" or "Other" title in would do the job, but I couldn't get it to work locally, so I'm simply reporting it.

Thanks,

Laurent

**Comment by jeremyevans** [comment]
Thanks for pointing that out.  I should probably add a rake task that checks that all plugins are documented on documentation.  I'll fix this issue before today's release, and work on a rake task after release.

**Reply by loranger32** [reply]
Hi Jeremy,

And thanks for the quick fix. I realize I made a mistake in the fix, I forgot to put text content int the <a> tag : host_authorization.

Sorry for the distraction.

Laurent
